otter/convergence/service.py

```diff
@@ -392,6 +391,7 @@ def convergence_succeeded(executor, scaling_group, group_state, resources,
         yield cf_msg('group-status-active',
                      status=ScalingGroupStatus.ACTIVE.name)
     # update servers cache with latest servers
+    now = yield Effect(Func(datetime.utcnow))
```

Comment:
Presumably the timing of this function call is pretty important.  When does it happen? Should the docstring say anything about that, or a comment at least?